pkg/doc/verifiable/credential_test.go

```diff
@@ -121,31 +121,34 @@ func TestNewCredential(t *testing.T) {
 
 func TestValidateVerCredContext(t *testing.T) {
 	t.Run("test verifiable credential with empty context", func(t *testing.T) {
-		raw := &rawCredential{}
+		var raw rawCredential
+
 		require.NoError(t, json.Unmarshal([]byte(validCredential), &raw))
 		raw.Context = nil
 		bytes, err := json.Marshal(raw)
 		require.NoError(t, err)
-		err = validate(bytes, nil, &credentialOpts{disabledCustomSchema: true})
+		err = validate(bytes, nil, &credentialOpts{})
```

Comment:
@troyronda pointed this out in the chat: why was this changed?

Reply:
It was just a redundant setting of the parameter. The majority of validation tests pass regardless of `disabledCustomSchema` option value.